compiler/src/org.graalvm.compiler.truffle.compiler.amd64/src/org/graalvm/compiler/truffle/compiler/amd64/substitutions/AMD64ArrayUtilsSubstitutions.java

```diff
@@ -113,7 +113,7 @@ private static int indexOfChar(char[] array, int arrayLength, int fromIndex, cha
 
     private static int indexOfChar(byte[] array, int arrayLength, int fromIndex, char[] chars) {
```

Comment:
rename to method to indexOfCharCompact. Move rename and move class to general location